cmd/chase.go

```diff
@@ -62,20 +63,12 @@ var chaseCmd = &cobra.Command{
 	Use:   "chase",
 	Short: "chase is used for auditing projects with OSS Index",
 	RunE: func(cmd *cobra.Command, args []string) (err error) {
-		logger = logrus.New()
-
-		switch verbose {
-		case 1:
-			logger.Level = logrus.InfoLevel
-		case 2:
-			logger.Level = logrus.DebugLevel
-		case 3:
-			logger.Level = logrus.TraceLevel
-		default:
-			logger.Level = logrus.ErrorLevel
+		logLady, err = getLogger(verbose)
```

Comment:
Good ole `logLady`
![](https://media.giphy.com/media/bR2tBgwxpEhTW/giphy.gif)

Reply:
NEVER FORGET THE LOG LADY